### DISCUSSION TITLE
Mirror dimension glitch?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7781] [QA - tech support + general] [answered]
Hi!!! So, uh, i was scrolling through the MASfandom subreddit, and i saw post showing monika gliched and i thought it was funny. how do u trigger the glitch?  ill put a image (IMAGES BELOW ARE NOT MINE ITS FROM THE SUBREDDIT)

![5i08sc14x2o71](https://user-images.githubusercontent.com/82217573/133824756-6051404e-b598-430c-8d19-60bc456a0704.png)
![2dpml668jyn71](https://user-images.githubusercontent.com/82217573/133824760-c14f29d3-b02d-45a3-bd0e-103ca864bfca.png)

those images i put are not mine for the 2nd time lol

**Comment by ThePotatoGuy** [comment, marked as answer]
not an intended glitch, and without a bug report, not worth an investigation.
